bluebrain/repo-bluebrain/packages/neurodamus-model/package.py

```diff
@@ -105,7 +123,7 @@ def merge_hoc_mod(self, spec, prefix):
         This routine simply adds the additional mods to existing dirs
         so that incremental builds can actually happen.
         """
-        core_prefix = spec['neurodamus-core'].prefix
+        core_prefix = spec['py-neurodamus'].prefix
```

Comment:
How does this work with things using with `version_from_model_core_dep`? Shouldn't `spec['py-neurodamus']` then just be `None`?

This bit seems the only problematic thing to me. IMO it would be cleaner to have PyDamus and HocDamus both providing a "virtual" NeuroDamus, and then use this virtual dependency to access stuff in the spec.

Of course, switching between the different providers would require a `^py-neurodamus` etc in the environment.

Reply:
Hum... broken backward compat. Your idea looks fancier but IMO we can get away with a simpler check for what dependency is available. Let me draft smtg for that